Do not strip the elf files to remove the warning

	NOTE: package openldap-2.4.31-r3: task do_populate_sysroot: Succeeded
	WARNING: File '/usr/bin/ldapcompare' from openldap was already stripped, this will prevent future debugging!
	WARNING: File '/usr/bin/ldapmodrdn' from openldap was already stripped, this will prevent future debugging!
	WARNING: File '/usr/bin/ldapmodify' from openldap was already stripped, this will prevent future debugging!

Signed-off-by: Roy.Li <rongqing.li@windriver.com>
---
 build/top.mk |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/build/top.mk
+++ b/build/top.mk
@@ -59,7 +59,7 @@ INSTALL_PROGRAM = $(INSTALL)
 INSTALL_DATA = $(INSTALL) -m 644
 INSTALL_SCRIPT = $(INSTALL)
 
-STRIP = -s
+STRIP =
 
 LINT = lint
 5LINT = 5lint
